src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -461,12 +462,14 @@ private static Properties readConfigFileProperties(final Path configFilePath) {
      * user can create their own data sources directory, which may not contain the metadata we seek.
      *
      * NOTE: The README file in a Data Sources directory is assumed to have the following properties:
```

Comment:
Should this now read:  `The MANIFEST file in a Data ....`?

Reply:
Yes.  Fixed!